.github/workflows/molecule-tests.yml

```diff
@@ -0,0 +1,124 @@
+---
+name: Molecule tests
+
+on:
+  push:
+  pull_request:
+
+env:
+  CARTRIDGE_CLI_VERSION: '2.4.0'
+
+jobs:
+  molecule-tests-ce:
+    if: |
+      github.event_name == 'push' ||
+      github.event_name == 'pull_request' && github.event.pull_request.head.repo.owner.login != 'tarantool'
```

Comment:
Why is this check needed

Reply:
To run tests for only one of PR or push, not both.

Reply:
This should solve it:


```yaml
on:
   push:
      branches: ['master']
   pull_request:
```

Reply:
We want to run workflow even if branch was pushed without PR.